### PR TITLE
Add Docker to the project

### PR DESCRIPTION
**Pull request by 4Vitalii5**
Requirements
Add Dockerfile
Add .env file
Add docker-compose.yml file. Use MySQL DB in the docker-compose.yml file.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    ports:
+      - "${SPRING_LOCAL_PORT}:${SPRING_DOCKER_PORT}"
+      - "${DEBUG_PORT}:${DEBUG_PORT}"
+    environment:
+      SPRING_APPLICATION_JSON: '{
+      "spring.datasource.url": "jdbc:mysql://mysqldb:${MYSQLDB_DOCKER_PORT}/${MYSQLDB_DATABASE}?serverTimezone=UTC",
+      "spring.datasource.username": "${MYSQLDB_USER}",
+      "spring.datasource.password": "${MYSQLDB_ROOT_PASSWORD}"
+      }'
+      JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
```
Comment (okuzan) [inline docker-compose.yml]: 5005 to DEBUG_PORT

**Review by okuzan — CHANGES_REQUESTED** [review]
let's create .env.template with actual values after = removed, .env is not supposed to be version controlled

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,13 +1,7 @@
...
-
 spring.jpa.hibernate.ddl-auto=validate
 spring.jpa.show-sql=true
 spring.jpa.open-in-view=false
 server.servlet.context-path=/api
 
-jwt.expiration=450000
-jwt.secret=HelloitsmeWooow1234216966521321426547657434123155654
+JWT.EXPIRATION=450000
+JWT.SECRET=HelloitsmeWooow1234216966521321426547657434123155654
```
Comment (okuzan) [inline src/main/resources/application.properties]: why uppercase?

Comment (4Vitalii5) [reply src/main/resources/application.properties]: I wanted to move the values ​​of these variables into the .env, but it didn't work for me either.

**Review by Elena-Bruyako — APPROVED** [review]

**Pull request merged**